src/picongpu/include/plugins/adios/ADIOSCountParticles.hpp

```diff
@@ -87,14 +91,17 @@ struct ADIOSCountParticles
 
     typedef Frame<OperatorCreateVectorBox, NewParticleDescription> AdiosFrameType;
 
-    HINLINE void operator()(ThreadParams* params,
-                            const std::string subGroup)
+    HINLINE void operator()(ThreadParams* params)
     {
         DataConnector &dc = Environment<>::get().DataConnector();
         GridController<simDim>& gc = Environment<simDim>::get().GridController();
         uint64_t mpiSize = gc.getGlobalSize();
         uint64_t mpiRank = gc.getGlobalRank();
 
+        const std::string speciesGroup( FrameType::getName() + std::string("/") );
```

Comment:
only a note: it is allowed to write ` const std::string speciesGroup( FrameType::getName() + "/" );`